src/app/public/modules/repeater/repeater.component.spec.ts

```diff
@@ -1448,4 +1452,32 @@ describe('Repeater item component', () => {
       flush();
     }));
   });
+
+  describe('with nested repeater items', () => {
+    let fixture: ComponentFixture<NestedRepeaterTestComponent>;
+    let el: any;
+
+    beforeEach(fakeAsync(() => {
+      fixture = TestBed.createComponent(NestedRepeaterTestComponent);
+      el = fixture.nativeElement;
+      fixture.detectChanges();
+      tick();
+    }));
+
+    it('should reorder top-level repeater items', fakeAsync(() => {
+      const selector = '.sky-repeater-item-group-1';
+      let topLevelItems = el.querySelectorAll(selector);
+      const initialBottomRepeaterItem = topLevelItems[topLevelItems.length - 1];
+
+      expect(initialBottomRepeaterItem).not.toBeNull();
```

Comment:
This strictly checks for not `null` and passed, but `expect(initialBottomRepeaterItem).toBeDefined()` fails. Perhaps the fixture could help with the selector.

Reply:
Well, it looks like my selectors weren't very reliable because they assumed this was the first and only test. I re-approached the selectors and based it on the tags so it won't be so brittle. This is ready for you again.